extensions-core/druid-kerberos/src/main/java/io/druid/security/kerberos/DruidKerberosModule.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.security.kerberos;
+
+import com.fasterxml.jackson.databind.Module;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Binder;
+import com.google.inject.Inject;
+import com.google.inject.multibindings.Multibinder;
+import com.metamx.http.client.HttpClient;
+import io.druid.guice.JsonConfigProvider;
+import io.druid.guice.LazySingleton;
+import io.druid.guice.annotations.Client;
+import io.druid.guice.annotations.Global;
+import io.druid.guice.http.HttpClientModule;
+import io.druid.guice.http.JettyHttpClientModule;
+import io.druid.initialization.DruidModule;
+import io.druid.server.initialization.jetty.ServletFilterHolder;
+import io.druid.server.router.Router;
+
+import java.util.List;
+import java.util.Properties;
+
+/**
+ */
+public class DruidKerberosModule implements DruidModule
+{
+
+  private static final String KERBEROS_ENABLED = "druid.authentication.kerberos.enabled";
+
+  @Inject
+  private Properties props;
```

Comment:
If i am not wrong the only reason this is needed is to read the value of `druid.authentication.kerberos.enabled`.
IMO this is not need since if the user chose to add this extension to the list of loaded module it means they want it activated and if they don't all they have to do is to remove it from the list. So i guess it is fair to assume that there is no need for this flag and turn it on by adding it to the list.